scripts/tests/Ice/background.py

```diff
@@ -2,4 +2,5 @@
 # Copyright (c) ZeroC, Inc. All rights reserved.
 #
 
-TestSuite(__name__, libDirs=["testtransport"], options={ "mx": [False] })
+# SSL doesn't work with this test when holding on reading on the server side
```

Comment:
It's not clear what "holding" means here. Is it possible to run a subset of this test with ssl, and skip a portion of the test based on the transport?

Reply:
The transport wrapper implemented by the test holds on reads: it doesn't call read on the underlying transport. It's possible we could make it work with SSL by disabling some tests but I'd rather not spend time on a test that most likely will have to be removed or completely reworked after the transport API refactoring.